src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 0) {
+    return 0;
+  }
+
+  if (days >= 7) {
+    return Number(days) * 40 - 50;
+  }
+
+  if (days >= 3) {
+    return Number(days) * 40 - 20;
+  }
+
+  if (days < 3) {
+    return Number(days) * 40;
```

Comment:
move "magic" numbers to variables.
I mean when someone looks at your code it's not clear what is 40, 20, 50?
also you can create a variable for days * 40.
and why do you need a Number(days)? days is already a number